src/exercises/bare-metal/rtc/src/pl031.rs

```diff
@@ -86,8 +86,8 @@ impl Rtc {
 
     /// Returns whether there is currently an interrupt pending.
     ///
-    /// This should be true iff `matched` returns true and the interrupt is
-    /// masked.
+    /// This should be true iff (if and only if) `matched` returns true and the
```

Comment:
Again, let's omit "iff" entirely and just write "if and only if".

Reply:
Okay, sure:

```suggestion
    /// This should be true if and only if `matched` returns true and the
```